oligo_designer_toolsuite/oligo_specificity_filter/_specificity_filters.py

```diff
@@ -0,0 +1,35 @@
+class SpecificityFilter:
+    """Thus class applies the specificity filters given in input sequentally to a ``CustomDB`` class.
```

Comment:
**This** class applies the specificity filters given in input **sequentially** to a ``CustomDB`` class.